pep257.py

```diff
@@ -778,6 +778,17 @@ def check_newline_after_last_paragraph(self, definition, docstring):
                                  'quotes on separate line')
 
     @check_for(Definition)
+    def check_whitespaces(self, definition, docstring):
+        """D210: White spaces around doctring should be trimmed."""
+        if docstring:
+            lines = eval(docstring).split('\n')
+            if lines[0].startswith(' ') or \
+                    len(lines) == 1 and lines[0].endswith(' '):
+                return Error(
```

Comment:
Put the line break in the string.

    return Error('D210: White spaces around docstring should '
                 'be trimmed.')